doc/dune-files.rst

```diff
@@ -1668,6 +1669,17 @@ The stanza will build all ``.v`` files on the given directory. The semantics of
   composition with the Coq's standard library is supported, but in
   this case the ``Coq`` prefix will be made available in a qualified
   way. Since Coq's lang version ``0.2``.
+- you can enable the production of Coq's native compiler object files
+  by setting ``<coq_native_mode>`` to ``native``, this will pass
+  ``-native-compiler on`` to Coq and install the corresponding object
```

Comment:
Sorry for only noticing this now, but you probably meant `yes` instead of `on`.

Reply:
Both are accepted, right? I can switch to `yes/no` if you want.

Reply:
I don't know / am not aware. I said this based on the output of `coqtop -help`.

Reply:
Yes they are [see for example the discussion on attributes where people advocated for more liberal parsing, but indeed here we see the documentation problem with that apporach]

I hadn't noticed that the documentation says so